gltf-importer/src/lib.rs

```diff
@@ -241,73 +152,58 @@ fn load_external_buffers(
         .collect()
 }
 
-fn validate_standard(json: &gltf::json::Root, config: &Config) -> Result<(), Error> {
+fn validate_standard(
```

Comment:
Not sure if this is the right one to export - I've thought about alternate importers (-> HTTP). What I imagined is one public function to import a .gltf or .glb from binary data, and another one to validate the result (all just for convenience of course, as far as I see it wouldn't be too much code as it is now).